apps/teams-test-app/src/components/PagesAPIs.tsx

```diff
@@ -156,6 +156,23 @@ const ReturnFocus = (): React.ReactElement =>
     },
   });
 
+const ReturnFocusWithType = (): React.ReactElement =>
+  ApiWithTextInput<pages.ReturnFocusType>({
+    name: 'returnFocusWithType',
+    title: 'Return Focus With Type',
+    onClick: {
+      validateInput: (input) => {
+        if (!input) {
```

Comment:
This should validate that input is one of your enum values

Reply:
addressed